src/theme/DocItem/Footer/styles.module.css

```diff
@@ -0,0 +1,93 @@
+.lastUpdated {
+  margin-top: 0.2rem;
+  font-style: italic;
+  font-size: smaller;
+}
+
+@media (min-width: 997px) {
+  .lastUpdated {
+    text-align: right;
+  }
+}
+
+.feedback_question {
+  font-size: 14px;
+  font-style: italic;
+  padding-right: 14px;
+  line-height: 40px;
+}
+
+.thanks {
+  margin-left: 0px;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.comment_box {
+  height: 150px;
+  padding: 12px;
+  margin-top: 4px;
+  margin-right: 16px;
+  margin-left: 16px;
+  box-sizing: border-box;
+  max-width: 80%;
+  border: 2px solid var(--ifm-color-primary);
+  border-radius: var(--ifm-alert-border-radius);
+  background-color: var(--ifm-navbar-background-color);
+  resize: none;
+}
+
+.feedback_button {
```

Comment:
Can we not reuse `snwpl-nav-button`? And only override a few things here. Otherwise this includes lots of inline colors that we will forget to change if we change the top buttons.